website/docs/roadmap.md

```diff
@@ -2,8 +2,20 @@
 
 We continuously work on updating our roadmap and we love to discuss those with our community. Feel encouraged to participate.
 
+## Q1 2024
+
+* 🎊 Supports embedding api for document context related use cases: https://github.com/TabbyML/tabby/issues/790
```

Comment:
Do we consider integrating this API into chat playground in Q1 as well? Once the API is ready, I suppose the integrations should be rather straightforward.

Reply:
This one is a bit tricky - it's still not clear how these context should be integrated.

One nice PoC built by sourcegraph folks is https://opencodegraph.org/ - I feel at least we could do some integration with our code browser (once it's implemented in #538).

The code browser could use some chat ui, probably.

Reply:
Cool let's research and discuss more in Q1 then.